### PR TITLE
[navX] add alpha channel to data

### PR DESCRIPTION
**Pull request by mathildemerle**
Small hack for https://github.com/Inria-Asclepios/music/pull/231

### CODE REVIEW

**Comment by LoicCadour** [comment]
Could you test the result on a NavX platform?

**Comment by mathildemerle** [comment]
I changed the system here, to include color AND opacity. I'm still waiting for Jatin to test the output files :)

File: src-plugins/vtkDataMesh/interactors/vtkDataMeshInteractor.cpp [hunk 1]
```diff
@@ -239,7 +240,37 @@ void vtkDataMeshInteractor::setupParameters()
...
+        double * color = d->metaDataSet->GetActor(0)->GetProperty()->GetColor();
+
+        int final_colors[3];
+        final_colors[0] = int(color[0]*255.0);
+        final_colors[1] = int(color[1]*255.0);
+        final_colors[2] = int(color[2]*255.0);
+
+        // decimal to hex color
+        QString colorHex;
+        colorHex += QString("#");
```
Comment (fcollot) [inline src-plugins/vtkDataMesh/interactors/vtkDataMeshInteractor.cpp]: You don't need to convert to QString each time:
```
colorHex += "#";
```

**Comment by fcollot** [comment]
looks good

**Comment by mathildemerle** [comment]
Idem as https://github.com/Inria-Asclepios/music/pull/231, Jatin replied.

**Pull request merged**